featurex/converters/api.py

```diff
@@ -29,6 +30,7 @@ def __init__(self, api_key=None):
         self.recognizer = sr.Recognizer()
         self.api_key = api_key
 
+    @abstractmethod
```

Comment:
Actually typing this out maybe making `environ_key` and `recognize_method` abstract would be the way to go